client/src/components/Modal/RegisterModal/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+// import "./styles.less";
+import { Modal, Button } from "antd";
+import { useDispatch, useSelector } from "react-redux";
+import Register from "../../../pages/Auth/Register/index"
+import {showRegisterModal, hideRegisterModal} from '../../../store/registerModalAction'
+
+function ModalRegister() {
```

Comment:
Use arrow functions instead of function declarations, as agreed.

Reply:
+1 agree